middleware/buffer/source/order.cpp

```diff
@@ -162,8 +163,12 @@ namespace casual
                   }
                   catch( ...)
                   {
-                     if( common::exception::capture().code() == common::code::xatmi::argument)
-                        return CASUAL_ORDER_INVALID_HANDLE;
+                     const auto error = common::exception::capture();
+
+                     if( error.code() == common::code::xatmi::argument)
+                        return CASUAL_ORDER_INVALID_HANDLE; 
+
+                     common::stream::write( std::cerr, "error: ", error, '\n');
```

Comment:
Why do we write to `std::cerr` this could happen not only in CLI context I presume?

Reply:
It's been like that for ages in CASUAL_FIELD, so it was just to make it coherent, but we can do whatever appropriate